common/libs/values.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import Big from 'big.js';
+import { toWei } from 'libs/units';
+
+// FIXME does this even makes sense
+export function toHex(n: Big): string {
```

Comment:
sure we do. https://github.com/kvhnuke/etherwallet/blob/e7d048233e52bea4b7d8b0b11fc4bb23c99f0dea/app/scripts/solidity/param.js